src/perf/lib/quicmain.cpp

```diff
@@ -19,18 +19,18 @@
 #include "quicmain.cpp.clog.h"
 #endif
 
-const MsQuicApi* MsQuic;
+const MsQuicApi* MsQuic = nullptr;
```

Comment:
Unnecessary, as is the one below. All globals are zero init'ed.

Reply:
Fixed